users/kpalatzky/kpalatzky_keymap_60.c

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2021 Kevin Palatzky
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _NAV 1
+#define _SYMB 2
+#define _NUM 3
+#define _MOUSE 4
+#define _ADJUST 5
```

Comment:
An enum should be used here, instead
```suggestion
enum custom_layers {
    _BASE,
    _NAV,
    _SYMB,
    _NUM,
    _MOUSE,
    _ADJUST,
};
```